Sources/muterCore/MutationOperators/RemoveSideEffectsOperator.swift

```diff
@@ -107,6 +107,31 @@ enum RemoveSideEffectsOperator {
     }
 }
 
+private extension FunctionDeclSyntax {
+    func hasImplicitReturn() -> Bool {
+        guard let body = body else {
+            return false
+        }
+        
+        return body.statements.count == 1 &&
+            signature.output != nil &&
+            signature.output?.isReturningVoid == false
+    }
+}
+
+private extension ReturnClauseSyntax {
+    var isReturningVoid: Bool {
+        returnType.withoutTrivia().description.trimmed == "Void" ||
```

Comment:
nitpick: doing `returnType.withoutTrivia().description.trimmed` twice. Could do `["Void", "()"].contains(...)`, or do a local variable.

Also, is there no more canonical way to get the type name than `description.trimmed`?

Reply:
Not that I'm aware of :(